src/app/containers/Include/index.jsx

```diff
@@ -23,14 +23,80 @@ const decodeHTML = (str) => {
   return str.replace(replacementsRegex, (match) => replacedParts[match]);
 };
 
-const IncludeContainer = ({ html, type }) => {
+const IncludeContainer = ({ html, type, href }) => {
   const { enabled } = useToggle('include');
+  // const [data, setData] = useState('');
+  const [includeHtml, setIncludeHtml] = useState('');
+  const [scriptTags, setScriptsTags] = useState([]);
 
   const supportedTypes = {
     idt2: 'idt2',
     vj: 'vj',
   };
 
+  const createAppendScriptTag = (code, src) => {
+    return new Promise((resolve, reject) => {
+      const script = document.createElement('script');
+      if (src) {
+        script.src = src;
+        // eslint-disable-next-line func-names
+        script.onload = function () {
+          resolve();
+        };
+      } else if (code) {
+        script.appendChild(document.createTextNode(code));
+      }
+      // eslint-disable-next-line func-names
+      script.onerror = function () {
+        reject();
+      };
+      document.body.append(script);
+      if (code) {
+        resolve();
+      }
+    });
+  };
+
+  useEffect(() => {
+    const fetchInclude = async () => {
+      const response = await fetch(href);
```

Comment:
We shouldn't be fetching in the includes container as this will mean includes are no longer be fetched serverside, we can just use the html that is passed into the include.

You may need to move your fetch into Storybook.